Slides: NumPy annual update

InessaPawson opened the pull request:
Adding the _NumPy annual update_ slides that were used for the presentation at the SciPy Tools plenary session.

InessaPawson committed:
Add info.json

InessaPawson committed:
Upload the presentation in pdf format

mepa commented on presentations/tools/numpy/info.json:
```diff
@@ -0,0 +1,12 @@
+    }
+  ],
+  "description": "Comprehensive overview of the latest release and major milestones 
+  for the NumPy library and its contributor community"
```
Hi! It looks like a newline is causing invalid json - could you please fix that?

InessaPawson committed:
Remove the new line

Remove the new line as requested in https://github.com/scipy-conference/scipy_proceedings/pull/1008#discussion_r1721174115
